NiceIO.Tests/ChangeExtension.cs

```diff
@@ -40,5 +40,17 @@ public void ToEmptyString()
 			var actual = new NPath("/my/path/file.something.exe").ChangeExtension(string.Empty);
 			Assert.AreEqual(expected, actual);
 		}
+
+		[Test]
+		public void BlockedOnWindowsRoot()
```

Comment:
i'll merge this pr, but can you rename these Blocked tests to be a bit more descriptive?